src/common/reconciler.go

```diff
@@ -355,18 +312,19 @@ func (r *ServiceReconciler) handleTools(service *opslevel.Service, registration
 		err := r.client.CreateTool(tool)
 		if err != nil {
 			log.Error().Msgf("[%s] Failed assigning tool '{Category: %s, Environment: %s, Name: %s}'\n\tREASON: %v", service.Name, tool.Category, toolEnv, tool.DisplayName, err.Error())
-		} else {
-			log.Info().Msgf("[%s] Ensured tool '{Category: %s, Environment: %s, Name: %s}'", service.Name, tool.Category, toolEnv, tool.DisplayName)
+			continue
 		}
+		log.Info().Msgf("[%s] Ensured tool '{Category: %s, Environment: %s, Name: %s}'", service.Name, tool.Category, toolEnv, tool.DisplayName)
 	}
 }
 
 func (r *ServiceReconciler) handleRepositories(service *opslevel.Service, registration opslevel_jq_parser.ServiceRegistration) {
 	for _, repositoryCreate := range registration.Repositories {
-		if repositoryCreate.Repository.Alias == nil || *repositoryCreate.Repository.Alias == "null" {
+		repoCreateString := "repoCreate{}"
+		if repositoryCreate.Repository.Alias == nil || *repositoryCreate.Repository.Alias == "" {
```

Comment:
JQ parser should not be returning this - if a repository does not have an alias it cannot be added. https://github.com/OpsLevel/opslevel-jq-parser/pull/44

Tests in the "happy path" however can so I am retaining this for now.